syncer/db.go

```diff
@@ -151,32 +152,38 @@ func (conn *Conn) executeSQLImp(sqls []string, args [][]interface{}) error {
 		txnHistogram.WithLabelValues(conn.cfg.Name).Observe(time.Since(startTime).Seconds())
 	}()
 
-	txn, err := conn.db.Begin()
-	if err != nil {
-		log.Errorf("exec sqls[%v] begin failed %v", sqls, errors.ErrorStack(err))
-		return errors.Trace(err)
-	}
-
-	for i := range sqls {
-		log.Debugf("[exec][sql]%s[args]%v", sqls[i], args[i])
+	for {
```

Comment:
`syncer` unit also has retry mechanism in `executeSQLJob`, and user can configure the max-retry count. So I think retry is not needed here.

Reply:
It' ok then. Let me close this pr